main.go

```diff
@@ -190,6 +229,15 @@ For more information, please go to https://github.com/brancz/kube-rbac-proxy/iss
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	var (
+		secureServingInfo    *server.SecureServingInfo
+		loopbackClientConfig *rest.Config
```

Comment:
godoc please.  This is weird to see here.  Dead? If so, can you make it up in the method you're calling?

Reply:
I am testing removing this now. I think we might not need the loopback support here. Will update based on result.

Reply:
As it turns out, loopback adds self-signed server certificate. This is needed when no certificate is provided from command line options.